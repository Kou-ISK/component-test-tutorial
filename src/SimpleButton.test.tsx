import { render } from '@testing-library/react';
import { SimpleButton } from './SimpleButton';

test('描画されてすぐはONと表示されている', () => {
  const view = render(<SimpleButton />);
  expect(view.container).toMatchSnapshot();
});
