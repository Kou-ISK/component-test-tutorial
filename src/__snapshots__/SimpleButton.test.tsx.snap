// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`描画されてすぐはONと表示されている 1`] = `
<div>
  <button>
    ON
  </button>
</div>
`;
